include/vapor/SliceRenderer.h

```diff
@@ -36,52 +37,73 @@ class RENDER_API SliceRenderer : public Renderer {
         int                 compressionLevel;
         int                 textureSampleRate;
         int                 orientation;
+        double              xRotation;
+        double              yRotation;
+        double              zRotation;
+        double              xOrigin;
+        double              yOrigin;
+        double              zOrigin;
         std::vector<float>  tf_lut;
         std::vector<double> tf_minMax;
         std::vector<double> boxMin, boxMax;
         std::vector<double> domainMin, domainMax;
         std::vector<double> sampleLocation;
     } _cacheParams;
 
+    struct _vertex {
```

Comment:
I admit that the use of _vertex is a little obscure.  It serves to map 3D vertices to 2D vertices within the slice plane.  Once we find the connectivity of the 2D vertices, we can know the 3D connectivity too.

Since _vertex is using both elements of the struct, I don't think a union makes sense.  Am I misunderstanding something?

Reply:
Sorry, my comment indeed does not make sense after your explanation but I was confused by this code: You have a struct called `_vertex` that contains two vertices, one 2D and one 3D, without any further explanation. It is not apparent to the reader what this is trying to accomplish. Since you are mapping from 3D coords to 2D coords you could use `map<vec3, vec2>` or `vector<pair<vec3, vec2>>` but maybe just a more descriptive name would suffice?

Reply:
Ok, I changed the name.  Also added a comment in the header since I think there's more to be described than can fit in a variable name.